static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
I guess the main argument for these being in templates/partials is that they can include django localization. Several of the modal partials do this.

If they're not using haml or any django tags... then maybe _static/templates_ is fine since that saves the app having to handle requests for them...

@ericnewcomer thoughts since you've done similar things in rapidpro?